friends/friendsController.js

```diff
@@ -0,0 +1,160 @@
+const router = require('express').Router();
+const friend = require('./friendsModel');
+
+// methods
+const errorMessage = (status, message, res) => {
+  res.status(status).json({
+    errorMessage: message
+  })
+  return;
+}
+
+router
+  .route('/')
+  .get((req, res) => {
+    friend.find()
+      .then(friends => {
+        // if (!friends.friends) {
+        //   errorMessage(404, 'friends could not be found', res);
+        // }
+        res.status(200).json(friends);
+      })
+      .catch(err => res.status(500).json({
+        error: 'Error making friends'
```

Comment:
This is a sad error 😆 "error making friends" - but I think it would be "error retrieving friends" because the database already exists.